src/HomeWorkApp.java

```diff
@@ -1,54 +0,0 @@
-public class HomeWorkApp {
-    public static void main(String[] args) {
-        printThreeWords();
-        checkSumSign();
-        printColor();
-        compareNumbers();
-    }
-
-    public static void printThreeWords(){
-        System.out.println("Orange");
-        System.out.println("Banana");
-        System.out.println("Apple");
-    }
-
-    public static void checkSumSign(){
-        int a = 10;
-        int b = -20;
-
-        int sum = a+b;
-
-        if (sum >= 0){
-            System.out.println("Сумма положительная");
-        } else {
-            System.out.println("Сумма отрицательная");
-        }
-    }
-
-    public static void printColor(){
-        int value = -50;
-
-        if (value <= 0){
-            System.out.println("Красный");
-        }
-        if (value >= 0 && value <= 100){
```

Comment:
```suggestion
        else if (value <= 100)
```
поскольку данное условие работает само по себе из-за неактульности первого `value<= 0 -> false <=> value > 0`